### ISSUE TITLE
Is it possible to use Vega instead of Vega-Lite?

### ISSUE THREAD

**Issue by tomthe** [idyll-lang/idyll-vega-lite#8]
I just played around with [Lyra 2](http://vis.csail.mit.edu/pubs/lyra2) (it is quite easy to create interactive charts with it) and wondered whether it is possible to include them into idyll documents. Maybe it would be good to keep the vega-json in a seperate file, as it becomes quite unwieldy.

**Comment by mathisonian** [comment]
Hi @tomthe I think it would be easy enough to duplicate this component to make a vega version, or to add an option to this one that tells it to drop-down into Vega. Is this something you'd be interested in contributing? 

Since the specs are just JSON you can use Idyll's data loading functionality to pass it in, something like:

```
[data name:"myVegaSpec" source:"my-vega-spec.json" /]

[IdyllVega spec:myVegaSpec /]
```

assuming you've created a file `data/my-vega-spec.json` in your idyll project.

**Comment by tomthe** [comment]
Hi @mathisonian, thank your for the hints. I never used React but this sounds easy enough for me to try.

**Comment by mathisonian** [comment]
Great! Let me know if you run into any issues and I can help resolve. I think it would be really cool to see a proof-of-concept of going from Lyra 2 to Idyll.

**Comment by danmarshall** [comment]
Funny I was just about to ask this same question :)
Since Vega-lite depends on Vega, we should essentially get this "for free". By looking at the `$schema` property, we know which type of spec it is.

Vega-lite:
```js
{
  "$schema": "https://vega.github.io/schema/vega-lite/v5.json"
}
```
Vega:
```js
{
  "$schema": "https://vega.github.io/schema/vega/v5.json"
}
```

**Comment by mathisonian** [comment]
For some reason just passing in a spec with the `$schema` set to vega wasn't working. 

However I pushed a small PR to support it: https://github.com/idyll-lang/idyll-vega-lite/pull/9. If you want to take a look at a suggested api & implementation to see if it suits your needs @tomthe @danmarshall

**Comment by danmarshall** [comment]
Thanks @mathisonian - I added a PR to your PR #10

**Comment by mathisonian** [comment]
This is now available starting in `v2.1.0` on npm
